### ISSUE TITLE
Need  a possibility to specify the name of state directory

### ISSUE THREAD

**Issue by zvnt** [datorama/akita#14]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
<pre><code>
 function buildPath( name, chosenDir ) {
    if(userPath) {
      return `${userConfig}/${chosenDir}/state/${name}`;
    }
     return `${process.cwd()}/${chosenDir}/state/${name}`;
}
</code></pre>
Currently the name of state directory is hardcoded in cli source code.

## Expected behavior
Please, add a possibility to specify the name of state directory :)

## What is the motivation / use case for changing the behavior?
I want a possibility to create state directory with specified name w/o "state" directory inside using cli.
e.g.
<pre><code>
/app/states/
    widgets/widgets.store.ts
    ....
    globals/globals.store.ts
    ...
    etc.
</code></pre>
Currently i must to do it manually.
Thx =)

**Comment by NetanelBasal** [comment]
Will do soon, thanks!

**Comment by NetanelBasal** [comment]
Released v1.4.0. You can now add the following config:

```json
"akitaCli": {
  "customFolderName": "true",
  "basePath": "./playground/src/app/"
}
```
